packages/cosmo-dycore/package.py

```diff
@@ -122,14 +122,14 @@ def cmake_args(self):
           cuda_arch = spec.variants['cuda_arch'].value
           if cuda_arch is not None:
               args.append('-DCUDA_ARCH=sm_{0}'.format(cuda_arch))
-          if '@gt2' in spec:
+          if '+gt1' in spec:
```

Comment:
I think the if/else should be reversed for gt1 which one should use gpu or CUDA ?